transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -235,6 +238,11 @@ public void showTransferMethodTypes(@NonNull List<TransferMethodSelectionItem> t
     }
 
     public void showCountryCode(final String countryCode) {
+        Map<String, String> params = new HashMap<>();
+        params.put("hyperwallet_ea_country", countryCode);
```

Comment:
When core insight is ready please use the class constant `InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_COUNTRY` as key